StandardLibrary/runtimes/python/pyproject.toml

```
@@ -16,7 +16,7 @@ include = [
 
 [tool.poetry.dependencies]
 python = "^3.11.0"
-pytz = "^2023.3.post1"
+pytz = "^2023.3.post1 || ^2024.1"
```

Hi @Ata-NovacekJan,

Thanks for your contribution!

Could you update your change to:

```suggestion
pytz = ">=2023.3.post1, <2025.0.0"
```

Your proposed expression would be complicated in (ex.) 2028:

```
pytz = "^2023.3.post1 || ^2024.1 || ^2025.0 || ^2026.0 || ^2027.0 || ^2028.0"
```

But the suggested expression would be simpler in 2028:

```
pytz = ">=2023.3.post1, <2029.0.0"
```

---

Hi, thank you.

Changed to `">=2023.3.post1, <2025.0.0"` as suggested.